Documentation/security/network/encryption-ipsec.rst

```diff
@@ -187,6 +193,16 @@ to all clusters in the mesh. You might need to increase the transition time to
 allow for the new keys to be deployed and applied across all clusters,
 which you can do with the agent flag ``ipsec-key-rotation-duration``.
 
+Encryption status
+=================
+
+Starting from Cilium CLI v0.15.23 cluster wide encryption status and IPsec key can be displayed using the following commands:
```

Comment:
This read like the key itself can be displayed, is it the case?

```suggestion
Starting from Cilium CLI v0.15.23, you can display cluster-wide encryption status and the status of the IPsec key with the following commands:
```

Reply:
Fixed, thanks!